aos/closed_loop/calcZernikesFromDonutStamps.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CalcZernikes from DonutStamps\n",
```

Comment:
<p>Does the usage of different dataset mean there are now 2 instead of 6 pairs of donuts available? Or has the donut selector not selected the other 4? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Yeah, I had to use a different field because the phosim field was actually too low to be realistically usable and while phosim would let you simulate it, imSim stops you from doing that.</p>